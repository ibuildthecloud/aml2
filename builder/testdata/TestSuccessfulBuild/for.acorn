for x in list {
	a: 1
}
for i, y in list2 {
	b: 2
}
