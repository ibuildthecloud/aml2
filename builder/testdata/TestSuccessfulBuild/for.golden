&eval.Struct{Fields: []eval.Field{
	&eval.Embedded{Expression: &eval.MergeObjectArray{
		Array: &eval.For{
			Value: eval.Value{
				Value: value.String("x"),
			},
			List: &eval.Lookup{
				Pos: eval.Position{
					Filename: "for.acorn",
					Offset:   9,
					Line:     1,
					Column:   10,
				},
				Key: "list",
			},
			Body: &eval.Struct{Fields: []eval.Field{&eval.KeyValue{
				Key: eval.Key{Value: eval.Value{
					Value: value.String("a"),
				}},
				Value: eval.Value{Value: value.Number("1")},
			}}},
		},
	}},
	&eval.Embedded{Expression: &eval.MergeObjectArray{Array: &eval.For{
		Key:   eval.Value{Value: value.String("i")},
		Value: eval.Value{Value: value.String("y")},
		List: &eval.Lookup{
			Pos: eval.Position{
				Filename: "for.acorn",
				Offset:   36,
				Line:     4,
				Column:   13,
			},
			Key: "list2",
		},
		Body: &eval.Struct{Fields: []eval.Field{&eval.KeyValue{
			Key:   eval.Key{Value: eval.Value{Value: value.String("b")}},
			Value: eval.Value{Value: value.Number("2")},
		}}},
	}}},
}}
